search.py

```diff
@@ -118,7 +121,8 @@ def solution(self):
         return [node.action for node in self.path()[1:]]
 
     def path(self):
-        """Return a list of nodes forming the path from the root to this node."""
```

Comment:
This repo doesn't follow flake8 completely. We put line limit to 100 characters if I remember correctly. Breaking a single, easy-to-read line
into multiple lines is no good.